cli/src/pcluster/schemas/cluster_schema.py

```diff
@@ -806,7 +806,7 @@ def make_resource(self, data, **kwargs):
 class QueueCustomActionSchema(BaseSchema):
     """Represent the schema of the custom action for the queue."""
 
-    script = fields.Str(required=True, metadata={"update_policy": UpdatePolicy.UNSUPPORTED})
+    script = fields.Str(required=True, metadata={"update_policy": UpdatePolicy.COMPUTE_FLEET_STOP})
```

Comment:
I decided to use the script as "update_key", it means that the script field cannot be updated.

If you're changing the script it means that you're inserting a new CustomAction and you will not see the differences in the update output. You'll see a deleted custom action and a new one.

The update of the CustomActions is already supported (see where the list of QueueCustomActionSchema is defined)

Reply:
Removed the commit for now